pkg/apply/applydrivers/apply_drivers_default.go

```diff
@@ -92,6 +95,11 @@ func (c *Applier) Apply() error {
 		c.ClusterDesired.CreationTimestamp = c.ClusterCurrent.CreationTimestamp
 	}
 	c.updateStatus(clusterErr, appErr)
```

Comment:
`appErr` is not used in `updateStatus`

Reply:
I will change func updateStatus and will use appErr to generate cmdPhase for each cmd.